src/query/storage/cache/redis_cache.go

```diff
@@ -142,17 +153,19 @@ func (cm CacheMetrics) CacheMetricsBucketHit(results []*storage.PromResult) {
 	}
 	cm.hitSamplesCounter.Inc(int64(tot_samples))
 	cm.hitBytesCounter.Inc(int64(tot_size))
+	cm.cacheBytesRetrieved.RecordValue(float64(tot_size))
 }
 
 // Update metrics for a cache miss
 func (cm CacheMetrics) CacheMetricsMiss(result storage.PromResult) {
-	// cm.missCounter.Inc(1)
+	cm.missCounter.Inc(1)
 	tot_samples := 0
 	for _, ts := range result.PromResult.Timeseries {
 		tot_samples += len(ts.Samples)
 	}
 	cm.missSamplesCounter.Inc(int64(tot_samples))
 	cm.missBytesCounter.Inc(int64(result.PromResult.Size()))
+	cm.m3dbBytesRetrieved.RecordValue(float64(result.PromResult.Size()))
```

Comment:
I think this is not per simple byte, this is per request bytes right?

Reply:
Yeah this metric is to record the number of bytes for each request. I already record the total number of bytes as a whole (missBytesCounter)